### PR TITLE
GEOS-7659: Updating ClassStatsPPIO to use empty string for namespaces.

### PR DESCRIPTION
**Pull request by jdeolive**

### CODE REVIEW

File: src/extension/wps/wps-core/src/test/java/org/geoserver/wps/gs/ClassificationStatsPPIOTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    StreamingSampleStats[] stats = new StreamingSampleStats[] {s1,s2};
+
+    ClassificationStats classStats = new FeatureClassStats.Results(ranges, stats);
+
+    ClassificationStatsPPIO ppio = new ClassificationStatsPPIO();
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    ppio.encode(classStats, bout);
+
+    System.out.println(new String(bout.toByteArray()));
```
Comment (aaime) [inline src/extension/wps/wps-core/src/test/java/org/geoserver/wps/gs/ClassificationStatsPPIOTest.java]: Forgotten a sysout here :-)

File: src/extension/wps/wps-core/src/test/java/org/geoserver/wps/gs/ClassificationStatsPPIOTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import static org.easymock.EasyMock.expectLastCall;
+import static org.easymock.EasyMock.notNull;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.verify;
+import static org.junit.Assert.assertEquals;
+
+public class ClassificationStatsPPIOTest {
+  
+  @Test
+  public void testSanity() throws Exception {
```
Comment (aaime) [inline src/extension/wps/wps-core/src/test/java/org/geoserver/wps/gs/ClassificationStatsPPIOTest.java]: LOL :-)

**Comment by jdeolive** [comment]
Removed the sysout, nice catch. Thanks for the review Andrea.

**Comment by jdeolive** [comment]
Travis is unhappy but I think it's an unrelated failure... 

    Failed tests:   testTripleChainedProgress(org.geoserver.wps.ExecuteTest): expected:<[38]> but was:<[25]>

**Comment by aaime** [comment]
Confirmed, this one builds for me. Merging

**Comment by jdeolive** [comment]
Thanks Andrea!

**Pull request merged**